curriculum/challenges/english/16-the-odin-project/top-learn-data-types-and-conditionals/learn-data-types-and-conditionals-question-a.md

```diff
@@ -0,0 +1,50 @@
+---
+id: 65e9713d484dd50f720e6fe6
+title: Learn Data Types and Conditionals Question A
+challengeType: 15
+dashedName: learn-data-types-and-conditionals-question-a
+---
+# --description--
+
+Depending on what kind of work you’re doing, you might end up working more with pieces of text rather than numbers. A **string** is a piece of text… and is a fundamental building block of the language.
```

Comment:
```suggestion
Depending on what kind of work you're doing, you might end up working more with pieces of text rather than numbers. A **string** is a piece of text… and is a fundamental building block of the language.
```

Very important